artifactory/services/security.go

```diff
@@ -124,6 +125,23 @@ func (ss *SecurityService) RevokeToken(params RevokeTokenParams) (string, error)
 	return string(body), err
 }
 
+// LookupTokenID looks up a token by username and return token id
+func (ss *SecurityService) LookupTokenID(username string) ([]string, error) {
```

Comment:
Let's rename the function name to more accurately reflect what is does. I suggest renaming it to *GetUserTokens*.
Let's also change the function's description and references in the tests accordingly.
In addition, let's add the new API to the README.